config/chunk-uploader.php

```diff
@@ -118,4 +118,49 @@
 
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Resumable.js Options
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure the options for the Resumable.js driver.
+    |
+    */
+
+    'resumable-js' => [
+
+        // The name of the multipart request parameter to use for the file chunk
+        'param' => 'file',
+
+        //  HTTP method for chunk test request.
+        'test-method' => \Illuminate\Http\Request::METHOD_GET,
+        //  HTTP method to use when sending chunks to the server (POST, PUT, PATCH).
+        'upload-method' => \Illuminate\Http\Request::METHOD_POST,
```

Comment:
```suggestion
        'upload-method' => Illuminate\Http\Request::METHOD_POST,
```

These aren't necessary when declaring them from the global namespace (which you are in in this file)